setup.py

```diff
@@ -30,10 +29,17 @@
     exit(1)
 
 
+# define macros for blkid releases
 macros = []
-
-if pkgconfig.installed("blkid", ">= 2.36"):
-    macros.append(("HAVE_BLKID2360", "1"))
+blkid_releases = ['2.36']
```

Comment:
The other required releases (*2.30* and *2.24*) should be added here too, right?

Reply:
Sorry, I seem to have pushed the wrong patch

Reply:
It's a bug and I will fix it